src/main/java/com/exasol/releaserobot/validation/AbstractPlatformValidator.java

```diff
@@ -29,7 +29,7 @@ protected void validateFileExists(final String filePath, final String fileDescri
         try {
             this.branchContent.getSingleFileContentAsString(filePath);
             this.validationReport.addSuccessfulValidation(fileDescription);
-        } catch (final GitHubException exception) {
+        } catch (final GitRepositoryException exception) {
```

Comment:
Similar as above, you might consider at some point create a more domain specific exception for this in a following PR, like "ValidationException" or similar.